grammar/rollem.pegjs

```diff
@@ -375,10 +380,10 @@ BasicRollConfiguration
     configuration.noSort = noSort ? true: false;
     return configuration;
   }
-
+//Implementing Critrange here. Prevents user from doing drop/keeps and specifying a range at the same time. 
```

Comment:
This may be inconvenient for 5e players that wish to specify advantage/disadvantage alongside critrange, but it can wait until they ask for it.